src/web/tools/InteractionManager.ts

```diff
@@ -97,10 +99,14 @@ export default class InteractionManager {
 
   public shouldHandlerBeCancelledBy(
     _handler: GestureHandler,
-    _otherHandler: GestureHandler
+    otherHandler: GestureHandler
   ): boolean {
-    //TODO: Implement logic
-    return false;
+    const shouldBeCancelledByScrollView =
```

Comment:
There's no actual check whether the `NativeViewGestureHandler` is attached to a scroll view. Either change the name or include the check, otherwise it may be confusing.